tasks/kafka-install.yml

```diff
@@ -13,6 +13,7 @@
   shell: gpg --verify {{ kafka_sig_tmp }} {{ kafka_bin_tmp }}
   ignore_errors: yes
   register: verify
+  changed_when: False
```

Comment:
I see, man that's such a pain! I wonder if it's the same in ansible v2..  At any rate, thanks for explaining!